src/CTA.WebForms2Blazor/FileConverters/ViewFileConverter.cs

```diff
@@ -66,23 +74,30 @@ private void FindConversionActions(HtmlNode node, HtmlNode parent)
 
         private void GetActions(HtmlNode node, HtmlNode parent)
         {
+            string controlConverterType = "";
             if (SupportedControls.ControlRulesMap.ContainsKey(node.Name))
             {
                 var conversionAction = new ControlConversionAction(node, parent, SupportedControls.ControlRulesMap[node.Name]);
+                controlConverterType = conversionAction.ControlConverter.GetType().Name;
                 _controlActions.Add(conversionAction);
-            }
+            } 
             else if (SupportedControls.UserControls.UserControlRulesMap.ContainsKey(node.Name))
             {
-                var conversionAction = new ControlConversionAction(node, parent,
-                    SupportedControls.UserControls.UserControlRulesMap[node.Name]);
+                var conversionAction = new ControlConversionAction(node, parent, SupportedControls.UserControls.UserControlRulesMap[node.Name]);
+                controlConverterType = conversionAction.ControlConverter.GetType().Name;
                 _controlActions.Add(conversionAction);
             }
             else
             {
-                // No conversion found found for this node type
-                // TODO: add metrics here if node.Name matches pattern: [A-z]*:[A-z]*
-                LogHelper.LogInformation($"No porting action found for node type: {node.Name}");
+                Match aspControlTagRegex = AspControlsRegex.Match(node.Name);
+                Match matchStartTagRegex = UnknownControlRemover.ControlStartTagRegex.Match(node.Name);
+                Match matchEndTagRegex = UnknownControlRemover.ControlEndTagRegex.Match(node.Name);
```

Comment:
In a similar vein to the comment above, the custom regex should be good enough for this check, `matchStartTagRegex` and `matchEndTagRegex` should never successfully match `node.Name`. Just for reference, the reason these regexes work later on when unsupported controls are removed is because they are compared against the entire view file content rather than a single node name